webapp/src/main/webapp/src/styles.less

```diff
@@ -139,6 +139,7 @@ footer {
 .my-drop-zone {
   border: dotted 3px lightgray;
   border-radius: 5px;
+  height: 118px; /* Required for drag & drop to work in IE11 */
```

Comment:
After the scrum call I tried hitting the ng2-file-upload site to see if the demo page's drag and drop was working in IE11, and to my astonishment, it was.  So I started investigating what was different between their implementation and mine, and found they were using a`.well` which applies a min-height to the element.